factory-containers/build.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/sh -e
```

Comment:
$ checkbashisms factory-containers/build.sh
possible bashism in factory-containers/build.sh line 5 (should be '.', not 'source'):
source $HERE/../helpers.sh
possible bashism in factory-containers/build.sh line 40 (${foo:3[:1]}):
TAG=${GIT_SHA:0:6}

If not changed to be generic, maybe force bash instead?

Reply:
its got to bin /bin/sh - docker:dind doesn't have bash.